editor/src/components/editor/remix-navigation-bar.tsx

```diff
@@ -45,6 +66,53 @@ export const RemixNavigationBar = React.memo(() => {
 
   const label = getRemixUrlFromLocation(navigationControls[EP.toString(activeRemixScene)]?.location)
 
+  const [currentURL, setCurrentURL] = React.useState<string | null>(label)
+
+  const addedToFavorites = currentURL != null && featuredRoutes.includes(currentURL)
+
+  React.useEffect(() => {
+    setCurrentURL(label)
+  }, [label])
+
+  const onInputChange = React.useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+    setCurrentURL(e.target.value)
+  }, [])
+
+  const changeRemixURLInEditor = React.useCallback(
+    (url: string) => {
+      try {
+        const baseDomain = `domainthatisveryunlikelytoberandomlyused.com`
```

Comment:
nice trick :D

Reply:
<img width="1184" alt="image" src="https://github.com/concrete-utopia/utopia/assets/2226774/f739b2a0-fd35-40be-b8dd-ef2dbbd6c9da">

🤔

Reply:
Stop it! :)